frontend/src/types/global.d.ts

```diff
@@ -0,0 +1,387 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+// with the License. A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+    ComputeFleetStatus,
+    InstanceState,
+    NodeType,
+    ImageBuildStatus,
+    ImageBuilderImageStatus,
+    Ec2AmiState,
+    ClusterStatus,
+    ValidationLevel,
+    CloudFormationStackStatus,
+    CloudFormationResourceStatus,
+    JobStateCode,
+} from './enums'
+
+
+export {}
```

Comment:
question: Is this somehow needed or just a leftover?

Reply:
Based on what I learned about type declarations, I believe this is needed to make the file a module. I added a comment.

Reply:
I just double-checked and this might not be needed so will try removing it.